style.css

```diff
@@ -1,3 +1,94 @@
 body {
-  background: yellow;
+  margin: 0;
+  font-family: sans-serif;
+  font-size: 1em;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.navbar {
+  display: block;
+  background: #2e84f6 ;
+  overflow:auto;
+  color: white;
+  font-size: .5rem;
+  font-family: "Source Sans Pro", sans-serif;
+
+}
+
+.navbar_title {
+  float: left;
+  padding-left: 1rem;
+
+}
+.navbar_portfolio {
+  float: right;
+  padding-right: 1rem;
+}
+
+.navbar_blog {
+  float: right;
+  padding-right: 1rem;
+}
+
+.navbar_contact {
+  float: right;
+  padding-right: 1rem;
+}
+
+.container {
+  max-width: 60rem;
+  margin: 0 auto;
+  padding: 1rem;
+  margin: 1rem;
+  display: inline-block;
+  width: calc(100% / 3 - 1rem - 1rem);
+}
+
+.profile_user {
+  overflow: auto;
+}
+
+.blog {
+    width: 25em;
```

Comment:
Use % widths for grids instead of hard values. Also, use `rem` instead of `em` values.